yeastphenome/apps/datasets/templates/genes/similar_genes.html

```diff
@@ -3,67 +3,83 @@
 {% load my_filters %}
 {% block title %}Similar Genes / {{ SITE_NAME }}{% endblock %}
 {% block content %}
-<style>
-#dataset-genes {
-  font: 10px sans-serif;
-}
+    <style>
```

Comment:
Same here - it's hard to see what is different (other than spacing!)

Reply:
Yeah, in this file it was auto-formatting and adapting the template to the changes in the view. It is all tested and does work locally.